### PR TITLE
Add IIT Associate Site

### PR DESCRIPTION
**Pull request by msherman64**
Add site config and nodes for IIT site.

Note: data is statically generated from hardware enrollment info, not yet live inventory data.

### CODE REVIEW

**Review by codyhammock — APPROVED** [review]
LGTM 👍🏻

File: data/chameleoncloud/sites/iit/iit.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "uid": "iit",
+    "name": "CHI@IIT",
+    "type": "site",
+    "site_class": "baremetal",
+    "description": "Illinois Institute of Technology",
+    "web": "https://chameleon.cs.iit.edu",
```
Comment (msherman64) [inline data/chameleoncloud/sites/iit/iit.json:7]: so, it turns out that it's really important that this line has no trailing `/`!

**Review by alexandru-orhean — APPROVED** [review]
LGTM

**Review by alexandru-orhean — APPROVED** [review]
LGTM

**Pull request merged**